apps/hermes/server/src/api/rest/v2/sse.rs

```diff
@@ -73,6 +73,10 @@ pub struct StreamPriceUpdatesQueryParams {
     /// If true, only include benchmark prices that are the initial price updates at a given timestamp (i.e., prevPubTime != pubTime).
     #[serde(default)]
     benchmarks_only: bool,
+
+    /// If true, invalid price IDs in the `ids` parameter are ignored. Default is `false`.
+    #[serde(default)]
+    ignore_invalid_price_ids: bool,
```

Comment:
I thought you said websockets are not affected by this?

Reply:
I guess this counts as SSE not WS? :D

Reply:
Yeah :)